lib/config/jquery.js

```diff
@@ -3,9 +3,7 @@ module.exports = {
     jquery: true,
   },
 
-  plugins: [
-    'shopify',
-  ],
```

Comment:
Are these changes the result of running prettier?

Reply:
Yup, it's within the 80 column size .